src/conf.py

```diff
@@ -300,3 +307,11 @@
 
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 #texinfo_no_detailmenu = False
+
+#
+# Get latest version of metachannel
+#
+BASEDIR = os.path.abspath(os.path.dirname(__file__))
+resp = requests.get('https://raw.githubusercontent.com/regro/conda-metachannel/master/README.md')
```

Comment:
You may want to add a `resp.raise_for_status()` after this line to make errors clearer. Otherwise, if this GET fails then you'll just write the error text into metachannel.md